mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -282,8 +282,10 @@ class CreateBlockRewrite : public BlockRewrite {
     block->dropAllUses();
     if (block->getParent())
       block->erase();
-    else
+    else {
```

Comment:
Nit: braces around the if when you need them for the else.

Reply:
done